.github/ISSUE_TEMPLATE/Outreachy-Challenge.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Outreachy Challenge
+about: An issue unrelated to badging submission
+title: Challenge: Open draft PR for Event Badge — @
+labels: 'onboarding', 'todo', 'Outreachy'
+assignees: ''
+
+---
+
+Go through the README to determine how to apply for an event badge at https://github.com/badging/diversity-and-inclusion/
+
+1. The goal is to open a “draft” Pull Request against master following the outlined process — feel free to creatively mock it like it is for a real event.
+
+   > **Hint** — This PR process emphasizes visibility and traceability by:
+   >
+   > - Appending a single row into the README.md
+   > - Retaining all application details and threads in a single op
+
+2. It helps to keep track of the steps you took in a secret gist, just the over all, nothing to detailed which can be found in the draft PR.
+
+   > **Hint** — Keeping notes in secret gists makes it possible to:
+   >
+   > - Keep `markdown` and other files together
+   > - Make revisions to them just like a repository
+   > - Share it with others using the secret link
+
+3. Place a link to this secret gist in a comment below. Please feel free to also keep notes on inconsistencies or things you felt could be improved — constructive feedback is a vital component of open collaboration.
+
+   > **Hint** — Collaboration comes in many forms, including:
+   >
+   > - Outlining friction
+   > - Analyzing problem(s)
+   > - Offering solution(s)
+
+4. Once you are done with the PR, place a link to to it in a comment below.
+
+---
```

Comment:
```suggestion
<!--

5. Make sure you complete the application process by following the steps here: 
https://github.com/chaoss/governance/blob/master/Outreachy-interest.md 

-->

```

Adding this can be helpful, especially if we leave it commented, so that we only show it once they are done with the chore.